leaks.go

```diff
@@ -23,10 +23,23 @@ package goleak
 import (
 	"errors"
 	"fmt"
+	"strings"
 
 	"go.uber.org/goleak/internal/stack"
 )
 
+// DefaultIgnoreFunctionSet can be set to a comma-separated list
+// of functions that may leak and should be ignored by goleak.
+// The registered set of functions will be then used similar to parameters
+// passed to [IgnoreAnyFunction]: any goroutine leaks with the given function
+// name(s) will be ignored by goleak.
+// This is helpful for library or tool authors that "owns" the leaking
+// goroutine and do not have explicit control over how the tests are run
+// by their consumers.
+// Unless you are in such a situation, you should be using [IgnoreAnyFunction],
+// [IgnoreCurrent], or [IgnoreTopFunction] instead.
+var DefaultIgnoreFunctionSet string
```

Comment:
Risk with making this an exported variable here is that libraries can't easily cooperate on setting this. It'll take one library to do `goleak.DefaultIgnoreFunctionSet = "foo"` to override other exceptions.